src/styles/style.scss

```diff
@@ -0,0 +1,49 @@
+
+.nav {
+  font-size: 12px;
+  text-transform: uppercase;
+
+  &__list {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+  }
```

Comment:
Why the file name is "style.scss"? It should be nav as the name of the BEM block.